compiler/luci/pass/src/FuseGeluPass.test.cpp

```diff
@@ -125,6 +125,32 @@ class FuseGeluTestGraph : public TestIOGraph, public GeluGraphlet
   }
 };
 
+class FuseGeluTestGraph2 : public TestIOGraph, public GeluGraphlet
```

Comment:
plz separate rename to FuseGeluTestGraph1 and adding this class.

Reply:
I will convert this PR, as a draft, and create PR with renaming and than PR with adding new class